covid19-etl/etl/idph.py

```diff
@@ -122,12 +143,14 @@ def parse_county(self, date, state, county_json):
 
         summary_location = {
             "country_region": country,
-            "county": county,
             "submitter_id": summary_location_submitter_id,
             "projects": [{"code": self.project_code}],
             "province_state": state,
         }
 
+        if county != "Illinois":
+            summary_location["county"] = county
```

Comment:
sorry but this "county" dance in this code is a bit confusing, would you mind adding a small comment, something like "the original data uses county="Illinois" for the aggregated IL state, but we use state="IL", county=null in our data"

Reply:
Makes sense, added comments.